strapi/strapi
# Fix password is sent when it was not changed

### What does it do?

Fix password is sent when it was not changed on the Edit user page.

### Why is it needed?

The password being an empty string leads to the validation error and inability to update the users.

| Validation error |
|--------|
| <img width="528" alt="Screenshot 2024-01-22 at 12 15 28 PM" src="https://github.com/strapi/strapi/assets/43180231/b8c3f9d2-e460-459c-9ac6-799611025594"> | 

### How to test it?

Try to update the user settings without changing the password and observe that no validation errors arise.
